sample-queries-tpch/tpch_query2.sql

```diff
@@ -1,4 +1,4 @@
-drop view q2_min_ps_supplycost;
+${EXPLAIN} drop view q2_min_ps_supplycost;

```

Comment:
This is not correct. The explain should be on the query not on the create definitions. This is true for all the queries that has multiple statements.